src/mgmt/srv_pool.c

```diff
@@ -58,6 +58,23 @@ ds_mgmt_tgt_pool_destroy_ranks(uuid_t pool_uuid, d_rank_list_t *filter_ranks)
 	return rc;
 }
 
+static uint32_t
+pool_create_rpc_timeout(crt_rpc_t *tc_req, size_t scm_size)
+{
+	uint32_t	timeout;
+	uint32_t	extra_timeout = 0;
+	int		rc;
+
+	rc = crt_req_get_timeout(tc_req, &timeout);
+	D_ASSERTF(rc == 0, "crt_req_get_timeout: "DF_RC"\n", DP_RC(rc));
+	if (bio_nvme_configured(SMD_DEV_TYPE_META)) {
```

Comment:
Maybe it's not necessary to check this, tmpfs in legacy mode has the exactly same issue. Probably we'd just simply bump pool create timeout no matter if it's md-on-ssd mode.